seed_test.go

```diff
@@ -0,0 +1,100 @@
+package migo
+
+import (
+	"database/sql"
+	"fmt"
+	"log"
+	"os/exec"
+	"testing"
+
+	"github.com/go-sql-driver/mysql"
+	"github.com/lestrrat/go-test-mysqld"
+)
+
+func TestSeedWithForeingKey(t *testing.T) {
+	conf := mysqltest.NewConfig()
+	conf.SkipNetworking = false
+	conf.Port = 13306
+
+	// start new instance of mysql
+	mysqld, err := mysqltest.NewMysqld(conf)
+	if err != nil {
+		t.Fatal("Failed to start mysqld:", err)
+	}
+	defer mysqld.Stop()
+
+	// connect to instance
+	sqlconf := mysql.Config{
+		User:   "root",
+		Addr:   "127.0.0.1:13306",
+		Net:    "tcp",
+		Passwd: "",
+		DBName: "",
+	}
+	db, err := sql.Open("mysql", sqlconf.FormatDSN())
+	if err != nil {
+		t.Fatal("Failed to connect to database:", err)
+	}
+	defer db.Close()
+
+	// create new database
+	query := fmt.Sprintf("CREATE DATABASE IF NOT EXISTS test")
```

Comment:
No need to use Sprintf function. simply define variable 
`query := "CREATE DATABASE IF NOT EXISTS test"`